FhirToDataLake/Synapse-E2E-Test.yml

```diff
@@ -0,0 +1,210 @@
+steps:
```

Comment:
To be consistent with current yaml file naming style, maybe we can change the file name to `synapse-e2e-test.yml`?

Reply:
Sure, renamed.